src/eventbuffer/InMemoryJSONEventBuffer.ts

```diff
@@ -425,53 +385,40 @@ export default class InMemoryJSONEventBuffer implements EventBuffer<EventData>,
         },
       ],
     };
-
-    let retryCount = 0;
     let failed = false;
     let response: Response = null;
     const body = this.makeRequestBody([event]);
     try {
-      do {
-        this.logger.debug(
-          `Event Reporting - InMemoryJSONEventBuffer - sendEventImmediately - body ${body} , retryCount ${retryCount}`
-        );
-        response = await this.send(body);
-        /* istanbul ignore else */
-        if (response.ok) {
-          try {
-            const data = await response.json();
-            this.logger.debug(
-              `Event Reporting - InMemoryJSONEventBuffer - sendEventImmediately - event send successful ${JSON.stringify(
-                data
-              )}`
-            );
-            return;
-          } catch (err) {
-            /* istanbul ignore next */
-            this.logger.debug(
-              `Event Reporting - InMemoryJSONEventBuffer - sendEventImmediately - Error reading OK response ${err}`
-            );
-          }
-        }
-      } while (
-        response &&
-        InMemoryJSONEventBuffer.SENDING_FAILURE_CODES.has(response.status) &&
-        ++retryCount < this.retryCountLimit
+      this.logger.debug(
+        `Event Reporting - InMemoryJSONEventBuffer - sendEventImmediately - body ${body}`
```

Comment:
Just want to confirm: do you really want to dump the entire event body in all log messages?

Reply:
Actually, it helps in debugging any failures for the rest of logs and does not need any mapping to do between the logs.

I can remove this one though, as I already print the received item at the top in this method.

Reply:
No, it's ok, I just wanted to check the intent. We can make the adjustments later.